.travis.d/definition_check.py

```diff
@@ -0,0 +1,67 @@
+import os
+import sys
+import re
+import fnmatch
+
+CURR_DIR = os.path.dirname(os.path.realpath(__file__))
+DEF_FILE_PATH = os.path.join(CURR_DIR, '..', 'docs', 'definitions.dox')
+INCLUDE_DIR = os.path.join(CURR_DIR, '..', 'include')
+
+DEFINE_RULE = re.compile(r' \*     \\defgroup ([a-zA-Z]+) [a-zA-Z ]+')
+USER_GROUP_RULE = re.compile(r' \* \\(user|kernel)group\{([a-zA-Z]+)\}')
+IGNORE_FILES = [
+    'vitasdk.h',
+    'vitasdk/utils.h',
+]
+
+# older python's glob not support `**`
+def findfile(directory, filepattern):
+    matches = []
+    for root, dirnames, filenames in os.walk(directory):
+        for filename in fnmatch.filter(filenames, filepattern):
+            matches.append(os.path.join(root, filename))
+    return matches
+
+def read_def_groups():
+    definitions = dict()
+    with open(DEF_FILE_PATH, 'r') as d:
+        for line in d.xreadlines():
+            m = DEFINE_RULE.match(line)
+            if not m:
+                continue
+            definitions[m.group(1)] = 0
+    return definitions
+
+def check_headers(definitions):
+    errors = []
+    # check exists in definitions
+    for header_path in findfile(INCLUDE_DIR, '*.h'):
+        header_file = header_path.split('include/')[1]
```

Comment:
Should we force headers in psp2 to use `\usergroup{}` and those in psp2kern to use `\kernelgroup{}`?